Source/Components/Row/MLBusinessRowView.swift

```diff
@@ -157,6 +157,12 @@ public class MLBusinessRowView: UIView {
         rightBottomInfo.translatesAutoresizingMaskIntoConstraints = false
         return rightBottomInfo
     }()
+    
+    private let blockedColor = MLStyleSheetManager.styleSheet.blackColor.withAlphaComponent(0.4)
+    private let closedBlackColor = "#737373".hexaToUIColor()
```

Comment:
Vince, cuando puedas pasate una captura para ver como queda con este color!

Reply:
Dale, pasó este ejemplo que hardcodee para que muestre todos los title y subtitle con el #737373 porque a esta hora están todos los locales abiertos y no encuentro ejemplos xD

![image](https://user-images.githubusercontent.com/49250268/124972161-6986dd00-e000-11eb-834a-c88a9caa36d4.png)

Reply:
La distancia queda más clara porque nos la manda BE. Al igual que el Envío, ambos debemos levantarlo en la reu de sync para que cuando el local esté cerrado nos envien #737373 👍🏼